README.md

```diff
@@ -0,0 +1,43 @@
+# terraform-aws-cloudtrail
+
+A Terraform module to create an Amazon Web Services (AWS) CloudTrail trail.
+
+## Usage
+
+This module creates a trail that logs to an S3 bucket. The module can be
+configured to log to an existing S3 bucket, or to make a new one for you
+automatically.
+
+```hcl
+module "cloudtrail" {
+  source = "github.com/azavea/terraform-aws-cloudtrail?ref=develop"
+
+  region                         = "us-east-1"
+  create_s3_bucket               = true
+  s3_bucket_name                 = "mysite-logs"
+  s3_bucket_lifecycle_expiration = 90
+
+  project     = "My Site"
+  environment = "Production"
+}
+```
+
+## Variables
+
+- `region` - Name of the region where the trail should be created (default:
+  `us-east-1`)
+- `create_s3_bucket` - Whether or not to create a new S3 bucket. When `false`,
+   you must provide a valid bucket to `s3_bucket_name` (default: `true`)
```

Comment:
This configuration is probably the most complicated part of the module. I wasn't sure if we wanted to support custom log buckets, but I figured it was safer to do so.

Reply:
Note that the module doesn't support CloudWatch logging in its current state. This is because I couldn't figure out how to allow a user to optionally enable it due to the [lack of undefined values in HCL](https://github.com/hashicorp/terraform/issues/5471) and the fact that the CloudWatch integration happens in an inline block on the `aws_cloudtrail` resource, not as a separate attachment (which means we can't use the `count` trick to enable or disable it).